examples/experimental/i3s-picking/app.js

```diff
@@ -147,7 +147,8 @@ export default class App extends PureComponent {
 
   _renderLayers() {
     const {tilesetUrl, token} = this.state;
-    const loadOptions = {throttleRequests: true};
+    // TODO: support compressed textures in GLTFMaterialParser
+    const loadOptions = {throttleRequests: true, i3s: {ignoreCompressedTextures: true}};
```

Comment:
Do you see good results from `throttleRequest`? Should we consider making it the default?

Maybe call the options `useCompressedTextures` and default to `true`?